src/main/java/com/mongodb/kafka/connect/source/MongoSourceConfig.java

```diff
@@ -250,56 +267,161 @@ public class MongoSourceConfig extends AbstractConfig {
           + "watched.";
   private static final String COLLECTION_DEFAULT = EMPTY_STRING;
 
-  public static final String COPY_EXISTING_CONFIG = "copy.existing";
+  public static final String STARTUP_MODE_CONFIG = "startup.mode";
+  private static final String STARTUP_MODE_CONFIG_DISPLAY =
+      "The behavior when there is no offset data.";
+  private static final String STARTUP_MODE_CONFIG_DOC =
+      format(
+          "A change stream cannot be resumed if there is no offset data."
+              + " In this case the connector may either ignore all/some existing source data,"
+              + " or may at first copy all existing source data and then continue with processing new data."
+              + " Possible values are %s."
+              + "\n- 'latest' is the default value. The connector ignores all existing source data and processes only new data."
+              + "\n- 'timestamp' actuates 'startup.mode.timestamp.*' properties."
```

Comment:
Add note that timestamp requires MongoDB 4.0 or greater.

Reply:
Done.